CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,9 @@
+<!--
+SPDX-FileCopyrightText: 2017 Scott Shawcroft, written for Adafruit Industries
+
+SPDX-License-Identifier: MIT
```

Comment:
This is incorrect. We based it on the Contributor Covenant which is CC-BY 4.0: https://www.contributor-covenant.org/ (at the bottom)

Also sourced from here: https://github.com/adafruit/Adafruit_Community_Code_of_Conduct

Reply:
Yeah this sounds like it needs to be updated. I'll update it in that repo, and also in TestRepo where this was overlooked.

Reply:
Sent https://github.com/adafruit/Adafruit_Community_Code_of_Conduct/pull/5 for tagging the code of conduct file.

Sent https://github.com/adafruit/Adafruit_CircuitPython_TestRepo/pull/34 for the test repo.

And updated on this one.